pkg/formatters/detailsFormatter_test.go

```diff
@@ -69,7 +69,8 @@ func TestDetailsFormatterNoDataReturnsTotalsCountAllZeros(t *testing.T) {
 	apiServerUrl := "https://127.0.0.1"
 
 	// When...
-	actualFormattedOutput, err := formatter.FormatRuns(runs, apiServerUrl)
+	formattableTest := NewDisplayableTestFromGalasaApi(runs, apiServerUrl)
```

Comment:
change the original test data to be of DisplayableTest rather than using the converter to change the format